.github/workflow/sync-automation.yml

```diff
@@ -0,0 +1,45 @@
+# Generated by: wf-gen from tyk-ci
+# Generated on: Wed  9 Dec 14:16:13 IST 2020
+
+
+# This workflow will silently ignore FILES that are missing in master
+# Most of the time this is what we want, older repos may not have some files
+
+name: Sync automation
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/*
+      - integration/*
+
+env:
+  FILES: .github/workflows/int-image.yml .github/workflows/del-env.yml integration/terraform/outputs.tf integration/image/Dockerfile .github/workflow/sync-automation.yml
+  
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        branch: [ release-2.9, release-3, release-3-lts, release-3.0.2, release-3.0.2-update, release-3.0.3, release-3.1, release-3.1.0, release-3.1.1, release-3.1.2 ]
```

Comment:
Can we just support all release-* branches?

Reply:
We could but the tradeoff is:
- a more complex script (`grep`, `sed`, `awk`, etc.)
- cannot use the matrix build to do everything in parallel

With this implementation, the limit of the automation is very clear. Moreover, these workflows are not maintained by hand, they are generated from templates. While reviewing, I noticed that the path of sync-workflow is wrong, pushing a fix.